modules/syslog-configuring-acs.adoc

```diff
@@ -1,14 +1,14 @@
-// Module included in the following assemblies:
+s// Module included in the following assemblies:
```

Comment:
Is this intial `s` supposed to be added here?

Reply:
Wow, nice catch, I will remove it :-)

Reply:
removed